lib/browser/objects-registry.js

```diff
@@ -20,6 +20,7 @@ class ObjectsRegistry {
   add(webContents, obj) {
     // Get or assign an ID to the object.
     let id = this.saveToStorage(obj);
+    this.storage[id].count++;
```

Comment:
The right sequence should be:

```
add(webContents1, 1)
remove(webContents, 1)
add(webContents1, 1)
get(webContents1, 1)
```

But due to the bug of `setDestructor` delaying the callback, it becomes:

```
add(webContents1, 1)
add(webContents1, 1)
remove(webContents, 1)
get(webContents1, 1)
```

That's how I see it.